kura/examples/org.eclipse.kura.example.gpio.led/OSGI-INF/metatype/org.eclipse.kura.example.gpio.led.LedExample.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2019 Eurotech and/or its affiliates
+
+     All rights reserved. This program and the accompanying materials
+     are made available under the terms of the Eclipse Public License v1.0
+     which accompanies this distribution, and is available at
+     http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+      Eurotech
+
+-->
+<MetaData xmlns="http://www.osgi.org/xmlns/metatype/v1.2.0" localization="en_us">
+    <OCD id="org.eclipse.kura.example.gpio.led.LedExample"
+        name="LedExample"
+        description="Example of switching on and off the led">
```

Comment:
"This example allows to configure and manage the state of a LED connected to a configurable GPIO pin"